municipal_finance/cubes.py

```diff
@@ -66,9 +66,13 @@ def get_manager():
             port,
             config['NAME'],
         )
+        # Setup connection arguments
+        connect_args = {
+            "options": "-c statement_timeout=30000"
```

Comment:
It'd be nice if this used the environment variable

Reply:
@jbothma done